useful_resources/computer_setup/python3_setup.md

```diff
@@ -0,0 +1,57 @@
+# Installing Python 3
+
+
+## Why Python 3?
+
+As you may have heard, starting [January 1, 2020][countdown] [Python 2.x is no
+longer officially supported][eol_article].
+
+Additionally, when using [Parsons][parsons], we use Python 3. Therefore, to make
+collaboration, debugging and maintenance easier, we encourage that you install and
+use Python 3 when writing scripts for TMC.
+
+
+## Getting Started
+
+There are a number of ways to [install Python 3][install_py3]. Before you
+install it, check that what version you already have.
+
+### Check Python Version
+
+1. Open the **Terminal** app
+2. Type `python --version`
+  * If you see something like `Python 3.x.x` then you are all set.
+    If not got to set 3
+3. Type `python3 --version`
+  * If you don't see any output then you need to install python 3.
```

Comment:
The rendering for the numbered list and then bullets looks a little funky. I'd consider trying something else.